.github/dependabot.yml

```diff
@@ -8,15 +8,15 @@ updates:
     schedule:
       interval: "daily"
     ignore:
-      - dependency-name: "edgenesis.io/shifu/k8s/crd" # ignore update for shifu modules
-      - dependency-name: "edgenesis.io/shifu/deviceshifu/pkg/mockdevice/mockdevice" # ignore update for shifu modules
+      - dependency-name: "github.com/edgenesis/shifu/pkg/k8s" # ignore update for shifu modules
+      - dependency-name: "github.com/edgenesis/shifu/pkg/deviceshifu/mockdevice/mockdevice" # ignore update for shifu modules
 
   - package-ecosystem: "gomod"
     directory: "k8s/crd" # Location of package manifests
     schedule:
       interval: "daily"
 
   - package-ecosystem: "gomod"
-    directory: "deviceshifu/pkg/mockdevice/mockdevice" # Location of package manifests
+    directory: "github.com/edgenesis/shifu/pkg/deviceshifu/mockdevice/mockdevice" # Location of package manifests
     schedule:
       interval: "daily"
```

Comment:
since now it's a mono repo, can we move dependabot for `k8s/crd` and `github.com/edgenesis/shifu/pkg/deviceshifu/mockdevice/mockdevice` ?  
I think the root go.mod should suffice